core/common/src/main/java/alluxio/clock/SystemClock.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.clock;
+
+/**
+ * The Java System clock.
+ */
+public class SystemClock implements Clock {
```

Comment:
a separate question: do we still stick to apply the `final` keyword when possible?